drive/src/fee_pools/epochs/operations_factory.rs

```diff
@@ -34,6 +34,10 @@ impl Epoch {
             .update_proposer_block_count_operation(proposer_pro_tx_hash, proposed_block_count + 1))
     }
 
+    pub fn add_init_empty_operations_no_storage(&self, batch: &mut GroveDbOpBatch) {
```

Comment:
According to our naming convention `add_*_operations` it should be `add_init_empty_without_storage_operations`